### PR TITLE
[#27] Multi-id resources and nested resources

### PR DESCRIPTION
**Pull request by blasterpal**
* also pending test for with_request nested url issues

### CODE REVIEW

File: spec/internal/app/models/vote.rb [hunk 1]
```diff
@@ -1,9 +1,10 @@
 class Vote
   include Mongoid::Document
+  include Matterhorn::Inclusions::InclusionSupport
```
Comment (Nakort) [inline spec/internal/app/models/vote.rb]: Why are we adding inclusion support and not using add_inclusion?

File: spec/internal/app/models/user.rb [hunk 2]
```diff
@@ -1,5 +1,6 @@
 class User
   include Mongoid::Document
+  include Matterhorn::Inclusions::InclusionSupport
```
Comment (Nakort) [inline spec/internal/app/models/user.rb]: Why are we adding inclusion support and not using add_inclusion?

Comment (blasterpal) [inline spec/internal/app/models/user.rb]: add_inclusion requires that on class. but the issue is not that I actually need inclusion. The problem is this line throws exception even though I'm not using the inclusion for a resource :index or :show: 

https://github.com/blasterpal/matterhorn/blob/27_multiid_get/lib/matterhorn/serialization/scoped.rb#L68

Without including the module the resource doesn't have `__inclusion_configs`. Could we possibly change this code, yes. Do you think we should open an issue and refactor this? I presumed ` Matterhorn::Inclusions::InclusionSupport` was required for models.

**Comment by Nakort** [comment]
+1

**Pull request merged**